docs/tutorial/persistable-classes.txt

```diff
@@ -0,0 +1,154 @@
+===================
+Persistable classes
```

Comment:
This can be "Type Maps and Persistable Classes" and I'd suggest renaming the file to `type-maps-persistable.txt` for SEO reasons, since that is used to derive the URI.

Alternatively, we can use another phrase like "Modeling BSON Data". That would set us up nicely to add additional sections down the line, such as a demonstration of the BSON interfaces (best use case there may be working with DateTime objects).

Reply:
Good idea, I renamed the file and changed the title to "Modeling BSON Data".